app/views/partials/foyer/individu-form.html

```diff
@@ -184,21 +180,24 @@
     </div>
   </div>
 
-  <div class="form-group from-group-lg" ng-if="captureTauxInvalidite()">
+  <div class="form-group from-group-lg" ng-if="captureTauxIncapacite()">
     <label class="control-label col-sm-4" for="eligibiliteAss">
-      Quel est {{ isDemandeur() ? 'votre' : 'son' }} taux d'invalidité ?
+      Quel est {{ isDemandeur() ? 'votre' : 'son' }} taux d'incapacité ?
+      <div class="help-block">
+        Évalué par <a target="_blank" href="http://informations.handicap.fr/carte-france-mdph.php">votre MDPH</a>.
+      </div>
     </label>
     <div class="col-sm-8">
       <label class="radio-inline">
-        <input type="radio" name="tauxInvalidite" value="moins50" ng-model="individu.tauxInvalidite">
+        <input type="radio" name="tauxIncapacite" value="moins50" ng-model="individu.tauxIncapacite">
         Moins de 50%
       </label>
       <label class="radio-inline">
-        <input type="radio" name="tauxInvalidite" value="moins80" ng-model="individu.tauxInvalidite">
+        <input type="radio" name="tauxIncapacite" value="moins80" ng-model="individu.tauxIncapacite">
         Entre 50% et 80%
       </label>
       <label class="radio-inline">
-        <input type="radio" name="tauxInvalidite" value="plus80" ng-model="individu.tauxInvalidite">
+        <input type="radio" name="tauxIncapacite" value="plus80" ng-model="individu.tauxIncapacite">
```

Comment:
> Since there is a default value, default to this one: always make the user actively _improve_ a life situation rather than having to actively describe it is worse than you expected.